src/samples/index.md

```diff
@@ -154,7 +154,7 @@ import 'package:test/test.dart';
 
 // Importing files
 import 'path/to/my_other_file.dart';
-import '../lib/samples/spacecraft.dart';
+import 'package:examples/samples/spacecraft.dart';
```

Comment:
This particular line is meant to be an example of an import statement that is using a path.

As a quickfix consider this change:

- Replace the `code-excerpt` instruction `replace` argument (in line 146) with this:
` replace="/'.*?' as \w+/'path\/to\/my_other_file.dart'/g; /package:examples/src/g"`

But we really should change the sample code so that we don't have to search-and-replace the path. After this PR is merged, I can make that change if you'd like.

Reply:
Or, I can make the update to this PR, let me know what you prefer.

Reply:
Please! Feel free to land it then. 😄

Reply:
Done